### DISCUSSION TITLE
Zähler voneinander subtrahieren

### DISCUSSION THREAD

**Discussion by Reborn2** [camueller/SmartApplianceEnabler#301] [Allgemein]
Hallo

gibt es eine Möglichkeit Zähler voneinander abzuziehen. Also z.B. den "Kühlschrank“ von der EG Wohnung. 

Dankeschön

**Comment by camueller** [comment]
Nein

**Reply by Reborn2** [reply]
Oh Schade, ist das für die Zukunft geplant. Wäre ja nur eine zusätzliche Option wie z.B der Umrechnungsfaktor.

**Reply by camueller** [reply]
Aktuell nicht geplant, da man entweder einen Zähler im Zählerschrank abfragt oder einen Steckdosenadapter wie Shelly Plug. Da braucht man keine Differenzen.

**Reply by Reborn2** [reply]
Okay Dankeschön, hätte halt gerne genau das gemacht Gesamtzähler der Wohnung um den Standby Verbrauch oder das Kochfeld abzufangen und davon bestehende Shelly’s abgezogen um nicht doppelt aufzuzeichnen.

**Reply by JoernHei** [reply]
Hallo camueller,
ich würde einfach Mal auch Bedarf für genau diese Funktion anmelden. Vielleicht ergibt sich ja eine Möglichkeit, wenn es mehrere Nutzer mit derselben Anfrage gibt.
Bei mir ist es ein Nebengebäude an dem ein E-Auto (mit eigenem Zähler des go-e Chargers) angeschlossen ist.

**Reply by camueller** [reply]
Der SAE verwaltet Geräte, die einen Zähler haben. Übergeordnete Zähler mit einer Vielzahl darunter befindlichen Geräte passen nicht in dieses Konzept und werden deshalb auch zukünftig nicht unterstützt.
